### PR TITLE
Update DDL descriptions in cluster upgrade

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Updates DDL descriptions when using smooth upgrade. This PR is a follow-up to https://github.com/pingcap/docs-cn/pull/18076.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/18076

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: upgrade-tidb-using-tiup.md [hunk 1]
```diff
@@ -171,7 +171,11 @@ tiup update cluster
 
 为避免升级过程中出现未定义行为或其他故障，建议检查以下指标后再进行升级操作。
 
-- 集群 DDL 情况：建议使用 [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) 命令查看集群中是否有正在进行的 DDL Job。如需升级，请等待 DDL 执行完成或使用 [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消该 DDL Job 后再进行升级。
+- 集群 DDL 情况：
+
+    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要等待 DDL job 完成或取消正在进行的 DDL job。
```
Comment (lilin90) [inline upgrade-tidb-using-tiup.md:176]:
```suggestion
    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。
```

File: upgrade-tidb-using-tiup.md [hunk 2]
```diff
@@ -171,7 +171,11 @@ tiup update cluster
...
+    - 如果你没有采用[平滑升级](/smooth-upgrade-tidb.md)，则建议使用 [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) 命令查看集群中是否有正在进行的 DDL job。如需升级，请等待 DDL 执行完成或使用 [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消该 DDL job 后再进行升级。
```
Comment (lilin90) [inline upgrade-tidb-using-tiup.md:177]:
```suggestion
    - 如果你没有采用[平滑升级](/smooth-upgrade-tidb.md)，则建议使用 [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) 命令查看集群中是否存在正在进行的 DDL job。如果存在，请等待 DDL 执行完成或使用 [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消该 DDL job 后再进行升级。
```

**Review by Frank945946 — APPROVED** [review]

File: upgrade-tidb-using-tiup.md [hunk 3]
```diff
@@ -171,7 +171,11 @@ tiup update cluster
 
 为避免升级过程中出现未定义行为或其他故障，建议检查以下指标后再进行升级操作。
 
-- 集群 DDL 情况：建议使用 [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) 命令查看集群中是否有正在进行的 DDL Job。如需升级，请等待 DDL 执行完成或使用 [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消该 DDL Job 后再进行升级。
+- 集群 DDL 情况：
+
+    - 如果你采用的是[平滑升级](/smooth-upgrade-tidb.md)，则不需要检查集群的 DDL 情况，无需等待 DDL job 完成或取消正在进行的 DDL job。
+    - 如果你没有采用[平滑升级](/smooth-upgrade-tidb.md)，则建议使用 [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) 命令查看集群中是否存在正在进行的 DDL job。如果存在，请等待 DDL 执行完成或使用 [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) 命令取消该 DDL job 后再进行升级。
```
Comment (qiancai) [inline upgrade-tidb-using-tiup.md:177]:
```suggestion
    - 如果你没有采用[平滑升级](/smooth-upgrade-tidb.md)，则建议使用 [`ADMIN SHOW DDL`](/sql-statements/sql-statement-admin-show-ddl.md) 语句查看集群中是否存在正在进行的 DDL job。如果存在，请等待 DDL job 执行完成或使用 [`ADMIN CANCEL DDL`](/sql-statements/sql-statement-admin-cancel-ddl.md) 语句取消该 DDL job 后再进行升级。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**